pkg/operator/ceph/cluster/mgr/mgr.go

```diff
@@ -47,38 +44,48 @@ const (
 
 var mgrNames = []string{"a", "b"}
 
-// Cluster is the ceph mgr manager
+// Cluster represents the Rook and environment configuration settings needed to set up Ceph mgrs.
 type Cluster struct {
+	// Configs from CRD (Alphabetical)
+	HostNetwork bool
 	Namespace   string
-	Version     string
 	Replicas    int
-	placement   rookalpha.Placement
-	context     *clusterd.Context
-	dataDir     string
-	HostNetwork bool
-	resources   v1.ResourceRequirements
-	ownerRef    metav1.OwnerReference
-	dashboard   cephv1beta1.DashboardSpec
+	Version     string
+
+	// Internal recordkeeping (alphabetical)
+	context   *clusterd.Context
```

Comment:
I found it confusing that some fields are capitalized and some are not. I think I just wanted badly for there to be some reasoning and thought perhaps the reasoning was if it was a setting from CRD. Is there a different reason some are capitalized and some aren't?

Reply:
The capitalization is actually a subtle feature of golang. Capital means that it will be public for other packages to see. Lower-case means it can only be used inside the package. So if a const/variable/func isn't needed outside the package it should be lower case.

Reply:
Are these used outside of this package?

Reply:
I haven't seen any usages of these capitalized params outside the `operator/cluster/mgr`. Should these be made lower case?

Reply:
I'll revert these back.